lib/lambda/processEmailsHandler.test.ts

```diff
@@ -276,3 +283,149 @@ describe("process emails  Handler failures", () => {
     await expect(() => handler(mockEvent, {} as Context, callback)).rejects.toThrow();
   });
 });
+
+const seatoolData = (authority: string) => ({
+  ACTION_OFFICERS: [
+    {
+      OFFICER_ID: 1,
+      FIRST_NAME: "OFFICER",
+      LAST_NAME: "ACTION",
+      EMAIL: "OFFICER_TEST@THISISFAKE",
+    },
+  ],
+  LEAD_ANALYST: [
+    { OFFICER_ID: 2, FIRST_NAME: "ANALYST", LAST_NAME: "LEAD", EMAIL: "ANALYST_TEST@THISISFAKE" },
+  ],
+  STATE_PLAN_SERVICETYPES: [
+    { SPA_TYPE_ID: 1, SPA_TYPE_NAME: "Medical" },
+    { SPA_TYPE_ID: 2, SPA_TYPE_NAME: "Dental" },
+  ],
+  STATE_PLAN_SERVICE_SUBTYPES: [
+    { TYPE_ID: 101, TYPE_NAME: "Emergency" },
+    { TYPE_ID: 102, TYPE_NAME: "Routine Checkup" },
+  ],
+  STATE_PLAN: {
+    SUBMISSION_DATE: 1704067200,
+    PLAN_TYPE: 122,
+    LEAD_ANALYST_ID: 45,
+    APPROVED_EFFECTIVE_DATE: 1706659200,
+    ACTUAL_EFFECTIVE_DATE: 1706659200,
+    PROPOSED_DATE: 1709251200,
+    SPW_STATUS_ID: 6,
+    STATE_CODE: "MD",
+    STATUS_DATE: 1709347600,
+    SUMMARY_MEMO: "Plan revision to include telehealth services.",
+    TITLE_NAME: "State Health Plan 2025",
+    CHANGED_DATE: 1709450000,
+  },
+  RAI: [
+    {
+      RAI_RECEIVED_DATE: 1704556800,
+      RAI_REQUESTED_DATE: 1704067200,
+      RAI_WITHDRAWN_DATE: 1704067201,
+    },
+  ],
+  ACTIONTYPES: [
+    { ACTION_ID: 10, ACTION_NAME: "Initial Review", PLAN_TYPE_ID: 3 },
+    { ACTION_ID: 11, ACTION_NAME: "Final Approval", PLAN_TYPE_ID: 3 },
+  ],
+  authority: authority,
+});
+describe("process emails  Handler for seatool", () => {
+  vi.spyOn(os, "updateData").mockImplementation(vi.fn());
```

Comment:
Seatool data structure is a bit differnt in what needs to be passed in so it needed its own function for looping through.